src/components/Box.tsx

```diff
@@ -1,9 +1,14 @@
-import PropTypes from 'prop-types';
 import { useContext } from 'react';
 import { ThemeContext } from '../context/themeContext';
 
-const Box = ({size, children}) => 
+interface Props {
+    size: string;
+    children: JSX.Element | JSX.Element[];
```

Comment:
This works in most cases but TS will be mad if you try to pass a string as children. `ReactNode` covers anything you could pass as `children`